fuzz/src/lib.rs

```diff
@@ -34,6 +37,44 @@ pub enum Creation<'a> {
     FromString(String),
     /// Create using `From<Box<str>>`, which consumes the `Box<str>` for `O(1)` runtime
     FromBoxStr(Box<str>),
+    /// Create from a number using [`ToCompactStr`]
+    NumToCompactStr(NumType),
+    /// Create from a boolean using [`ToCompactStr`]
+    BoolToCompactStr(bool),
+    /// Create from a char using [`ToCompactStr`]
+    CharToCompactStr(char),
+    /// Create from a String using [`ToCompactStr`]
+    StringToComapctStr(String),
+}
```

Comment:
Note that we also have specialisation for `CompactStr` in `ToCompactStr`.

Reply:
Good callout!